Framework/Muon/src/RemoveExpDecay.cpp

```diff
@@ -148,11 +148,12 @@ void MuonRemoveExpDecay::exec() {
 HistogramData::Histogram MuonRemoveExpDecay::removeDecay(
     const HistogramData::Histogram &histogram) const {
   HistogramData::Histogram result(histogram);
+  const auto xPointRawData = histogram.points().rawData();
```

Comment:
Sorry these two points are a bit pedantic...
- Please could you change this line so it reads the x points off the result object. The code used to read the bin edges from result and the Y and E values are also read off result so seems more consistent
- Also the Points class returned by points() already supports indexing using [i] so no need for the .rawData() bit on the end. Probably makes sense to rename the variable to xPoints or similar as part of this